cmd/policy/policy.go

```diff
@@ -143,7 +143,9 @@ func uploadCommand() *cobra.Command {
 				return err
 			}
 			if len(m.Policies) == 0 && !allowEmpty {
-				return fmt.Errorf("no policies found, use --allow-empty to upload no policies")
+				return fmt.Errorf("no policies found" +
+					"\n\t - Ensure path provided points to the parent directory of the /policies directory" +
+					"\n\t - or use --allow-empty to upload no policies")
```

Comment:
not sure we should allow no policies, https://lacework.atlassian.net/browse/RAIN-51435 prevents this on api server as we had a bug scenario. That would be a separate issue though removing this flag.